map/src/i18n/langs/tr.json

```diff
@@ -1,5 +1,6 @@
 {
   "title": "COVID-19 Karşılıklı Yardımlaşma Haritası",
+  "inFrameTitle": "Bu haritayı {map} üzerinde göster",
```

Comment:
```suggestion
```

adding translations directly in these json files will be removed by crowdin, we need to make sure that translations are only added using that UI (i.e. after the new english strings are added to `master`)